mockative/build.gradle.kts

```diff
@@ -11,6 +11,9 @@ kotlin {
         // Common
         named("commonMain") {
             kotlin.srcDirs("$buildDir/generated/mockative-code-generator")
+            dependencies {
+                implementation(kotlin("reflect"))
+            }
```

Comment:
This is an elegant solution. I like it! Could we move this dependency to the `jvmMain` source set, which seems to be the only place where this is needed?

Reply:
Oh yeah, absolutely.

Oversight when implementing it 😅

Doing it in the next minutes